index.html

```diff
@@ -32,32 +59,43 @@ <h3 class="SD">I'm a Software Developer</h3>
           <a href="https://twitter.com/OfficialAseT"><img src="/images/twitter.png" class="contact-icon"></a>
           <a href="#"><img src="/images/medium.png" class="contact-icon"></a>
         </section>
-    </div>
+  </div>
 </section>
 
     <section class="Recent-works">
-
+      <section class="recent-works-heading">
         <h1 class="recent-works-title">My Recent Works</h1>
-      
+        <div class="line"></div>
```

Comment:
When I looked at the desktop version, `My Recent Work` section
![image](https://user-images.githubusercontent.com/61361037/213739051-ccfc01e1-58ab-4a95-9f05-40b584c3f0ca.png)
It is behind your `intro` section. Kindly try to meet the FIGMA design requirement.
The purpose of this project is to learn how to successfully switch your design from mobile to desktop design using media queries.

Reply:
Hi @RaoAkif 

Kindly clarify this comment as I dont quite understand what you meant.